arcgis-ios-sdk-samples/Utility network/Display content of utility network container/DisplayContentUtilityNetworkContainerViewController.swift

```diff
@@ -0,0 +1,296 @@
+    
+    // The symbols used to display the container view contents.
+    let boundingBoxSymbol = AGSSimpleLineSymbol(style: .dash, color: .yellow, width: 3)
+    let attachmentSymbol = AGSSimpleLineSymbol(style: .dot, color: .blue, width: 3)
+    let connectivitySymbol = AGSSimpleLineSymbol(style: .dot, color: .red, width: 3)
+    
+    /// The action that is prompted when exiting the container view.
+    @IBAction func exitContainerView() {
+        // Disable the bar button item since container view will be exited.
+        exitBarButtonItem.isEnabled = false
+        // Remove all the objects that were added onto the graphics overlay.
+        (mapView.graphicsOverlays[0] as? AGSGraphicsOverlay)?.graphics.removeAllObjects()
+        (mapView.map?.operationalLayers as? [AGSLayer])?.forEach { layer in
+            // Make each operational layer visible.
+            layer.isVisible = true
+            guard let previousViewpoint = previousViewpoint else { return }
+            // Return to the viewpoint before container view was entered.
+            mapView.setViewpointGeometry(previousViewpoint.targetGeometry) { [weak self] _ in
+                // Enable interaction on the map view.
+                self?.mapView.isUserInteractionEnabled = true
+            }
+        }
+    }
+    
+    /// Create a map.
+    ///
+    /// - Returns: An `AGSMap` object.
+    func makeMap() -> AGSMap {
+        let webMapURL = URL(string: "https://ss7portal.arcgisonline.com/arcgis/home/item.html?id=5b64cf7a89ca4f98b5ed3da545d334ef")!
+        let map = AGSMap(url: webMapURL)!
+        return map
+    }
+    
+    /// Create and load the utility network using the feature service URL.
+    func loadUtilityNetwork() {
+        utilityNetwork = AGSUtilityNetwork(url: featureServiceURL, map: mapView.map!)
+        utilityNetwork?.load { [weak self] error in
+            if let error = error {
+                self?.presentAlert(error: error)
+            }
+        }
+    }
+    
+    /// Get the legend information provided by the feature layers used in the utility network.
+    func getLegends() {
+        // Create feature tables from URLs.
+        let electricDistributionTable = AGSServiceFeatureTable(url: URL(string: "https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer/105")!)
+        let structureJunctionTable = AGSServiceFeatureTable(url: URL(string: "https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer/900")!)
+        // Create feature layers using the feature tables.
+        featureLayers = [electricDistributionTable, structureJunctionTable].map(AGSFeatureLayer.init)
+        featureLayers.forEach { layer in
+            // Get the legend information of each layer.
+            layer.fetchLegendInfos { [weak self] (legendInfos, error) in
+                guard let self = self, let legendInfos = legendInfos else { return }
```

Comment:
We can first disable the legend button in storyboard, and enable the legend button here. 

Per my test, if there is no network connection, the legend table will only contain the 3 images for the attachment and box symbols.